templates/web/pages/about.tmpl

```diff
@@ -0,0 +1,135 @@
+{{ define "about" }}
+{{ template "header" . }}
+
+
+<section class="hero is-medium has-background-image" style='background-image: linear-gradient(170deg, rgba(0,0,0,0.7), rgba(155,155,200,0.7)), url({{ static_path "about_home_header.jpg" }})'>
+	<div class="hero-head">
+		{{ template "navbar" }}
+	</div>
+	<div class="hero-body">
+	</div>
+</section>
+
+<section class="section">
+	<div class="container">
+		<div class="columns is-centered">
+			<div class="column is-6">
```

Comment:
Add [content](https://bulma.io/documentation/elements/content/) as a class to this, then you can get rid of those `<br>`s in the the `<p>` tags.